tests/model_explainability/trustyai_service/service/multi_ns/test_trustyai_service_multi_ns.py

```
@@ -0,0 +1,222 @@
+
+
+    def test_drift_metric_delete_with_db_storage(
+        self,
+        admin_client,
+        minio_data_connection_multi_ns,
+        current_client_token,
+        trustyai_service_with_db_storage_multi_ns
+    ):
+        for tai in trustyai_service_with_db_storage_multi_ns:
+            verify_trustyai_service_metric_delete_request(
+            client=admin_client,
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)